logstash-logging-spring-boot-starter/src/main/java/pl/piomin/logging/filter/SpringLoggingFilter.java

```diff
@@ -54,13 +56,21 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
             final long startTime = System.currentTimeMillis();
             final SpringRequestWrapper wrappedRequest = new SpringRequestWrapper(request);
             if (logHeaders)
-                LOGGER.info("Request: method={}, uri={}, payload={}, headers={}, audit={}", wrappedRequest.getMethod(),
+                if(ignorePayload)
```

Comment:
Could you simplify it a little bit and not to have such many `if`? Because it's another rule we can create a builder object, and then just call withHeaders, withPayload...